Automattic/jetpack
# Moving jetpack_is_mobile into a package

- [x] Needs tests.

#### Changes proposed in this Pull Request:

* Moves `jetpack_is_mobile` into a separate package.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* This is decoupling a feature from Jetpack.

#### Testing instructions:






* Make sure `jetpack_is_mobile` is still usable and returns the same values as before.
* Make sure `Automattic\Jetpack\Mobile::is_mobile` works.

#### Proposed changelog entry for your changes:

* Move `jetpack_is_mobile` method into a separate package.
